src/utils/wallet.ts

```diff
@@ -211,7 +212,7 @@ const wallet = {
    * @memberof Wallet
    * @inner
    */
-  getAddresses(xpubkey: string, startIndex: number, quantity: number, networkName: string = 'mainnet'): Object {
+  getAddresses(xpubkey: string, changeDerivation: number, startIndex: number, quantity: number, networkName: string = 'mainnet'): Object {
```

Comment:
I feel I've already seen a similar change in this PR. Why do we need the `changeDerivation` here?